test/keyboard-layout.test.js

```
@@ -0,0 +1,148 @@
+/**
+ * @vitest-environment jsdom
+ * @vitest-environment-options { "url": "http://localhost/" }
+ */
+
+import { describe, it, expect, vi, beforeEach } from 'vitest';
```

ESLint

[error] 'beforeEach' is defined but never used.

(unused-imports/no-unused-imports)